content/articles/2024-05-01-tiles-based-logs.md

```diff
@@ -0,0 +1,37 @@
+
+After years of working with these logs at scale – building, operating, learning from other operators – we've learned there are better ways to design and implement APIs to make logs more scalable, particularly to reads, and cheaper to operate, in terms of engineering and storage resources. This tiles-based approach has already been tested and proven in several ecosystems, and has been available in our [serverless tooling](https://github.com/transparency-dev/serverless-log) since 2021. The ecosystems include the [Go module checksum database](https://go.dev/blog/module-mirror-launch), [ArmoryDrive Firmware Transparency](https://github.com/usbarmory/armory-drive/wiki/Firmware-Transparency), [Pixel Binary Transparency](https://developers.google.com/android/binary_transparency/pixel), and [Sunlight](https://letsencrypt.org/2024/03/14/introducing-sunlight.html) in Certificate Transparency. Today, we're announcing that we're working on a general, ecosystem-agnostic implementation which embodies these changes and explaining why we believe it's better for transparency ecosystems to adopt.
+
+## What are tiles?
+
+A Merkle tree is composed of hash values. To optimize storage and hash re-computation, Trillian splits the Merkle tree data into tiles, where a tile contains more than one hash value.
+
+![How to represent a Merkle tree with tiles](/images/tiles-based-logs/merkle-tree-tiles.svg)
+
+When Trillian serves a read request such as an [inclusion proof](https://transparency.dev/verifiable-data-structures/), it does some computing to find which hash values within a Merkle tree must be used to compute and prove a given entry is included in the tree. It fetches these hashes from tiles, and serves them to clients. The reader then takes the hash values the server provides, uses them to recompute the root hash, and checks that it matches the one claimed by the log.
+
+In a world where the interface to the log is via the tiles API, the log makes these subtree tiles available directly to clients, and rather than asking the log to build proofs, the clients identify which tiles includes the hashes they need and fetch them in order to build and verify the proof themselves.
+
+## Benefits of tiles
+
+So you can see that while the interface to the log is changing, the underlying cryptography — the hashing and proofs — are not. Why, then, are we making this change?
+
+*   Improve cacheability: Static, immutable resources, like tiles, are much easier to trivially cache via CDN, or even locally on the client, than parameterised requests as in Trillian.
```

Comment:
I think you can drop the "as in Trillian" bit here - the RPC Trillian API was never exposed to the world so we're really comparing cacheability of a tile-based API with other personality APIs like CT/Sigstore/etc.